build_zip

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
 
-# just a quick thing to rebuild the zip file. Update the paths as needed.
+# just a quick thing to rebuild the zip file.
+source build_config
 
-printf "\nRebuilding zip file.\n"
+# test if $NAME is not blank and file/directory $NAME does not exist
+if [ "$NAME" != "" -a ! -e "$NAME" ] ; then
+	# create a temporary directory
+	cp -R ./view/ ./$NAME
 
-rm 01PITT_INST-ULS_1_8.zip
-zip -r 01PITT_INST-ULS_1_8.zip 01PITT_INST-ULS_1_8/
+	# build a zip file
+	printf "\nRebuilding zip file.\n"
+	rm ${NAME}.zip
+	zip -r ${NAME}.zip ./${NAME}/
+	printf "\nRebuild complete.\n"
 
-printf "\nRebuild complete.\n"
+	# remove the temporary directory
+	rm -R ./${NAME}/
+else
+	echo "Error: the temporary directory we want to create already exists."
```

Comment:
When spitting out error messages, best practice is to write to STDERR rather than STDOUT.  You can use `>&2 echo something` for that.

Reply:
Oh! Yes, of course. Updated...